lib/__tests__/index.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+
+const assert = require('assert');
+const del = require('del');
+const fs = require('fs');
+const metalToolsSoy = require('../../index');
+const sinon = require('sinon');
```

Comment:
We can also just use the [mocks](https://facebook.github.io/jest/docs/en/mock-function-api.html#content) provided by jest